Dockerfile

```diff
@@ -7,6 +20,7 @@ ENV GIT_HASH $GIT_HASH
 
 RUN mkdir /leonardo
 COPY ./leonardo*.jar /leonardo
+COPY --from=helm-go-lib-builder /helm-go-lib-build/helm-scala-sdk/helm-go-lib /leonardo/helm-go-lib
```

Comment:
shall we copy just the shared lib?

Reply:
We could but the other files are minuscule compared to `libhelm.dylib`. Not sure if worth the removal. Was also thinking `go.{mod,sum}` files could be useful for reference?